core/primitives/src/serialize.rs

```diff
@@ -76,6 +78,34 @@ pub trait BaseDecode: for<'a> TryFrom<&'a [u8], Error = Box<dyn std::error::Erro
     }
 }
 
+macro_rules! pretty_bytes_serialization {
```

Comment:
Why is it called `pretty`? Should it be `base64_bytes_serialization`?

Reply:
Well, this is a reasonable comment, though, my thinking around this is that base64 is just an implementation detail, and the intent is to be able to serialize bytes into a "pretty format".